mlir/lib/Conversion/ArithToSPIRV/ArithToSPIRV.cpp

```diff
@@ -607,6 +607,35 @@ struct UIToFPI1Pattern final : public OpConversionPattern<arith::UIToFPOp> {
   }
 };
 
+//===----------------------------------------------------------------------===//
+// IndexCastOp
+//===----------------------------------------------------------------------===//
+
+/// Converts arith.index_cast to spirv.Select if the source type is i1
```

Comment:
```suggestion
/// Converts arith.index_cast to spirv.Select if the source type is i1.
```

See https://llvm.org/docs/CodingStandards.html#commenting